pkg/environment/environment.go

```diff
@@ -95,28 +93,26 @@ func IsUsingMTLS() bool {
 	return strings.ToLower(os.Getenv(UseMTLS)) == "true"
 }
 
-// GetCertSecretName returns the name of the kubernetes secret which holds the
-// manager/connectors.
-func GetCertSecretName() string {
-	return os.Getenv(CertSecretName)
-}
-
-// GetCertSecretNamespace returns the namespace of the kubernetes secret which holds the
-// manager/connectors.
-func GetCertSecretNamespace() string {
-	return os.Getenv(CertSecretNamespace)
-}
-
-// GetCACERTSecretName returns the name of the kubernetes secret that holds the CA certificates
-// used by the client/server to validate the manager to the manager/connectors.
-func GetCACERTSecretName() string {
-	return os.Getenv(CACERTSecretName)
-}
-
-// GetCACERTSecretNamespace returns the namespace of the kubernetes secret that holds the CA certificate
-// used by the client/server to validate the manager to the manager/connectors.
-func GetCACERTSecretNamespace() string {
-	return os.Getenv(CACERTSecretNamespace)
+// GetMinTLSVersion returns the minimum TLS version that is acceptable.
+// if not provided it returns zero which means that
+// the system default value is used.
+func GetMinTLSVersion(log *zerolog.Logger) uint16 {
```

Comment:
I'd suggest something like:
```go
func GetMinTLSVersion(log *zerolog.Logger) uint16 {
	minVersion := os.Getenv(MinTLSVersion)
	rv := uint16(0)
	switch minVersion {
	case "TLS-1.0":
		rv = tls.VersionTLS10
	case "TLS-1.1":
		rv = tls.VersionTLS11
	case "TLS-1.2":
		rv = tls.VersionTLS12
	case "TLS-1.3":
		rv = tls.VersionTLS13
	default:
		log.Info().Msg("MinTLSVersion is set to the system default value")
		return rv
	}
	log.Info().Msg("MinTLSVersion is set to " + minVersion)
	return rv
}
```

the strings "TLS-1.0" - "TLS-1.3" can be defined as constants, but it is not crucial

Reply:
Changed. Thanks